### PR TITLE
Stream Proof, visible check and more visuals

### PR DESCRIPTION
**Pull request by sestain**

### CODE REVIEW

**Review by michaellrowley — COMMENTED** [review]
You adjust main.cpp and the 'vcxproj' file, the vcxproj adjustment decreases the windows target platform which doesn't sound like a great idea.
The 'main.cpp' adjustment doesn't really contribute anything either.
Imo everything else looks fine!

File: CSGOSimple/main.cpp [hunk 1]
```diff
@@ -83,7 +83,12 @@ BOOL WINAPI OnDllDetach()
     return TRUE;
 }
 
-BOOL WINAPI DllMain(_In_      HINSTANCE hinstDll, _In_      DWORD     fdwReason, _In_opt_  LPVOID    lpvReserved) {
+BOOL WINAPI DllMain(
```
Comment (michael7845) [inline CSGOSimple/main.cpp:86]: I'm not the original dev of this project & I don't have any influence but I don't see the value in changing this.

File: CSGOSimple.DotSettings [hunk 2]
```diff
@@ -6,4 +6,4 @@
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CppFormatting/PLACE_ELSE_ON_NEW_LINE/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CppFormatting/PLACE_WHILE_ON_NEW_LINE/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CppFormatting/USE_INDENT_FROM_VS/@EntryValue">False</s:Boolean>
-	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EFeature_002EServices_002ECpp_002ECodeStyle_002ESettingsUpgrade_002ENamespaceIndentationSettingsUpgrader/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (michael7845) [inline CSGOSimple.DotSettings:9]: Is there any reason why this is necessary?

File: CSGOSimple/CSGOSimple.vcxproj [hunk 3]
```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{F3E42845-8D56-4BB3-821D-8163AB1337F0}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>CSGOSimple</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```
Comment (michael7845) [inline CSGOSimple/CSGOSimple.vcxproj]: Is there any reason why any adjustments to this file are necessary?

**Review by michaellrowley — COMMENTED** [review]
I think there still are still unnecessary changes here.

- CSGOSimple/CSGOSimple.vcxproj

- CSGOSimple.DotSettings

- CSGOSimple/main.cpp

File: CSGOSimple.DotSettings (hunk 2 above)
Comment (sestain) [reply CSGOSimple.DotSettings:9]: ReSharper did it automatically and Im not good at using github

File: CSGOSimple/main.cpp (hunk 1 above)
Comment (sestain) [reply CSGOSimple/main.cpp:86]: I changed it back to original but it accidently left spaces

File: CSGOSimple/CSGOSimple.vcxproj (hunk 3 above)
Comment (sestain) [reply CSGOSimple/CSGOSimple.vcxproj]: no

File: CSGOSimple.DotSettings (hunk 2 above)
Comment (sestain) [reply CSGOSimple.DotSettings:9]: no

**Review by sestain — COMMENTED** [review]
I changed all useless changes back to original and now it should be ready

**Pull request closed without merging**